src/Main.java

```diff
@@ -1,7 +1,63 @@
+import java.util.Arrays;
+
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-
+        System.out.println("Task01");
+        int[] array1=new int[] {1,2,3};
+        double[] array2=new double[] {1.57, 7.654, 9.986};
+        int[] array3=new int[33];
+        System.out.println("Task02");
+        for (int index=0; index<array1.length; index++) {
+            System.out.print(array1[index]);
+            if (index < array1.length-1) {
+                System.out.print(",");
+            }
+        }
+        System.out.println("");
```

Comment:
```suggestion
        System.out.println();
```
зачем пустую строку передавать?